Sources/Tests/UnitTests/Core/Settings.cs

```diff
@@ -64,8 +64,8 @@ public void AssertGetBackDespiteDispose()
         [Theory]
         [InlineData("x2=16")]
         [InlineData("x2 + 4x +4 = 0")]
-        [InlineData("x2 3")]
-        [InlineData("x2 0.5")]
+        [InlineData("2 3")]
+        [InlineData(") 2")]
```

Comment:
This one is not going to pass 😄 . Pig meant something like `(a + 2) 2`

Reply:
Classic web editing without thinking